R/render.R

```diff
@@ -64,6 +64,8 @@
 #' @references \url{http://naruto.wikia.com/wiki/Tsukuyomi}
 #' @importFrom htmltools tagList tags htmlEscape HTML
 #' @export
+#' @examples
+#' # rmarkdown::render('foo.Rmd', 'xaringan::moon_reader')
```

Comment:
@pat-s Next time please make sure to rebase against the lastest master branch here to avoid spurious changes like this in a PR:

```bash
git pull --rebase git@github.com:yihui/xaringan.git master
# then force push to your master branch
```

I don't recommend you to use your master branch for pull requests, because all commits will be squashed before merging, e.g. c2c2055d00c9c94c2bf32fafb32d9d9899573b91 Then you will have some trouble with syncing your master branch with the master branch here (your choices are (1) delete your repo and fork again; or (2) force push; either option is not good). For pull requests, I recommend that you always use different branches for different pull requests.

Reply:
Right, sorry for this. Usually I do it like this but for this PR I apparently did not! Next time again :+1: